docs/search/criteria_reference/contentname_criterion.md

```diff
@@ -0,0 +1,41 @@
+# ContentName Criterion
+
+The [`ContentName` Search Criterion](https://github.com/ibexa/core/blob/main/src/contracts/Repository/Values/Content/Query/Criterion/ContentName.php)
+searches for content by its name.
+
+## Arguments
+
+- `value` - string(s) representing the Content name(s), the joker `*` can be used for partial search.
+
+## Example
+
+### PHP
+
+``` php
+$query->query = new Criterion\ContentName(['laptop', 'tablet']);
```

Comment:
Seems the mentioned criterion accepts a single string only: https://github.com/ibexa/core/blob/main/src/contracts/Repository/Values/Content/Query/Criterion/ContentName.php#L16

Reply:
So, an example illustrting the joker, and that match phone and smart phone.
```suggestion
$query->query = new Criterion\ContentName('*phone');
```